styles/widgets/common/gallery.less

```diff
@@ -2,7 +2,7 @@
     width: 100%;
     height: 100%;
     .user-select(none);
-    .touch-action(pinch-zoom);
+    .touch-action(pinch-zoom pan-y);
```

Comment:
We have different widgets with this style. Did you check all of them?

Reply:
Most part of they are navigation widgets like Panorama or Pivot. Grid parts that have this style should allow Drag'n'Drop and restrict scrolling at the same time.

Reply:
Ok, Thanks for your explanation